src/store/filters.ts

```diff
@@ -215,6 +262,16 @@ export class Filters extends VuexModule {
         return false;
       }
 
+      // Annotation ID filters
+      const matchesAnnotationIds =
+        enabledAnnotationIdFilters.length === 0 ||
```

Comment:
Not important: `!enabledAnnotationIdFilters.length`

Reply:
Cool, I'll leave as is because I think it's every so slightly more readable.